### PR TITLE
chore: adds engine and chore source code

### PR DESCRIPTION
**Pull request by ematipico**
This PR does:
- replaces `==` with `===` in the source code;
- adds `engines` to the `package.json`. This matches what we have in `actions.yml`;
- adds a PR template ;
- fixes a potential bug where the promise emitted from `main` is not handled; 
- minify the code during the `build` step

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -199,5 +199,9 @@ function _getTempDirectory() {
 module.exports = main;
 
 if (require.main === module) {
-	main();
+	main()
```
Comment (MichaReiser) [inline src/index.js:202]: Does node support top level await? You could then use `await main()`;

File: package-lock.json [hunk 2]
```diff
@@ -1,286 +1,8 @@
 {
   "name": "setup-rome",
   "version": "0.0.1",
-  "lockfileVersion": 2,
```
Comment (MichaReiser) [inline package-lock.json:4]: Are these changes intentional?

File: src/index.js (hunk 1 above)
Comment (ematipico) [reply src/index.js:202]: node 16 does support top level away, either way the promise must be handled otherwise node kills the program

File: package-lock.json (hunk 2 above)
Comment (ematipico) [reply package-lock.json:4]: it changed the lock file because I updated npm to the latest version (8)

**Review by MichaReiser — APPROVED** [review]
